frontend/src/Context/PublisherProvider/usePublisher/usePublisher.tsx

```diff
@@ -76,8 +77,12 @@ const usePublisher = (): PublisherContextType => {
   const [isPublishing, setIsPublishing] = useState(false);
   const publisherOptions = usePublisherOptions();
   const [isForceMuted, setIsForceMuted] = useState<boolean>(false);
-  const [isVideoEnabled, setIsVideoEnabled] = useState(!!publisherOptions.publishVideo);
-  const [isAudioEnabled, setIsAudioEnabled] = useState(!!publisherOptions.publishAudio);
+  const [isVideoEnabled, setIsVideoEnabled] = useState<boolean>(
+    publisherOptions?.publishVideo ?? false
+  );
+  const [isAudioEnabled, setIsAudioEnabled] = useState<boolean>(
+    publisherOptions?.publishAudio ?? false
+  );
```

Comment:
@maikthomas, I followed your suggestion here but I still need to do the `useEffect` below. otherwise the publisher in the meeting room initialized with `false` and not what's in the `publisherOptions`

Reply:
mmm, in that case is this doing anything? or is `publisherOptions` always undefined in this scenario?

Reply:
I think `publisherOptions` is always `undefined` when we set the initial state of this. that's why I was setting it to `false` originally